cumulusci/utils/yaml/cumulusci_yml.py

```diff
@@ -35,7 +39,18 @@ def _replacer_func(matchobj):
 
 
 def cci_safe_load(f_config: IO[Text]):
-    "Load a file, convert NBSP->space and parse it in YAML."
+    """Load a file, convert NBSP->space and parse it in YAML.
+    Raises CumulusCIException if an error occurs while parsing.
+    """
     data = _replace_nbsp(f_config.read())
-    rc = yaml.safe_load(StringIO(data))
+    try:
+        rc = yaml.safe_load(StringIO(data))
+    except ScannerError as e:
```

Comment:
Looking at the PyYAML source, there are a few classes of exceptions which extend MarkedYAMLError. I think we probably want to catch that one here instead of only ScannerError.